src/actions/bookshelf/utils/bookshelf-utils.js

```diff
@@ -124,3 +127,63 @@ export function appendWhereClause (queryBuilder, fieldName, value, matchStrategy
       throw new Error(`Unrecognized match strategy "${matchStrategy}"`)
   }
 }
+
+// -----------------------------------------------------------------------------
+// Accepts the "orderBy" API field value to apply the appropriate
+// "queryBuilder.orderBy" logic.
+//
+// The function requires including:
+// joint        - The joint instance
+// queryBuilder - The queryBuilder instance
+// modelName    - The model name of the main resource
+// -----------------------------------------------------------------------------
+// This logic is supports ordering by columns of associations (via dot
+// notation). If the association is not defined on the source model, the orderBy
+// argument is ignored.
+//
+// NOTE: NULLS are always returned last in both ASC and DESC orders.
+// -----------------------------------------------------------------------------
+export function appendOrderByClause (joint, queryBuilder, modelName, fieldValue) {
+  const orderBy = buildOrderBy(fieldValue)
+
+  orderBy.map(orderOpt => {
```

Comment:
`.forEach()` is more appropriate here because no new array is being created.

Reply:
yup, yup!